test/ex_doc/autolink_test.exs

```diff
@@ -183,7 +220,13 @@ defmodule ExDoc.AutolinkTest do
 
       assert_unchanged("mix compile.elixir --verbose")
 
-      assert_unchanged("mix unknown.task")
+      captured =
+        assert_warn(fn ->
+          assert_unchanged("mix unknown.task")
+        end)
+
+      assert captured =~
+               "documentation references module \"unknown.task\" but it is undefined or private"
```

Comment:
I don't think we should warn here, we want to write docs that have e.g. `` `mix myapp.mytask` ``.

However, if we write `` [foo](`mix myapp.mytask`) `` then we are referring to a particular mix task so if it doesn't exist, we should warn. I also think, the warning should be slightly different, I think one of the below:

- `documentation references "mix unknown.task" (...)`
- `documentation references mix task "unknown task" (...)`
- `documentation references module "Mix.Tasks.Unknown.Task" (...)`

but we don't need to worry about the warning message too much right now imho.

Reply:
I have taken care of this already.